bindings/node/test/clientEncryption.test.js

```diff
@@ -592,222 +592,4 @@ describe('ClientEncryption', function () {
         });
     });
   });
```

Comment:
These tests were duplicates of tests in the driver (https://github.com/mongodb/node-mongodb-native/blob/4aa9d9d1add0a469d3c3c16ad70e574830158db2/test/integration/client-side-encryption/client_side_encryption.prose.test.js#L1509).

More importantly, the spec states explicitly that these tests must not be run against a standalone server.  We don't have the infrastructure to run anything other than a replica set in libmongocrypt (at least in the node bindings).

Additionally, these tests are skipped in CI because we only run tests that do not require a live server in CI.